From 1f385ff1bde1e783eba03df37af9421bac0df82b Mon Sep 17 00:00:00 2001
From: Peter Jones <pjones@redhat.com>
Date: Fri, 3 Aug 2018 09:56:01 -0400
Subject: [PATCH 28/34] abignore: work around an abidw bug

Some versions of abignore mistakenly treat brackets as section headers,
thus ignoring the rest of the stanza after them.  It's fixed upstream,
but I don't have the newer version yet.  Moving the braces to be after
the change-kind and other things works around the issue.

Signed-off-by: Peter Jones <pjones@redhat.com>
---
 src/abignore | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/src/abignore b/src/abignore
index a4597b1029c..1badb2dab35 100644
--- a/src/abignore
+++ b/src/abignore
@@ -14,28 +14,28 @@
 #               'efidp_wifi __anonymous_union__::wifi' at efivar-dp.h:868:1
 #
 [suppress_type]
-  soname_regexp = libefi(var|boot)\\.so\\..*
   name = efidp_data
   type_kind = typedef
   has_data_member_inserted_at = end
+  soname_regexp = libefi(var|boot)\\.so\\..*
 
 # 1 Added variable:
 #
 #  'const __anonymous_struct__ efi_guid_ux_capsule'    {efi_guid_ux_capsule@@LIBEFIVAR_1.33}
 #
 [suppress_variable]
+  change_kind = added-variable
   soname_regexp = libefivar\\.so\\..*
   symbol_name_regexp = ^efi_guid_[[:alnum:]_]+$
-  change_kind = added-variable
 
 # allow new functions that are prefixed correctly
 [suppress_function]
+  change_kind = added-function
   soname_regexp = ^libefi(var|boot)\\.so\\..*
   name_regexp = ^(efidp_|efi_)[[:alnum:]_]+
-  change_kind = added-function
 
 # allow new variables that are prefixed correctly
 [suppress_variable]
+  change_kind = added-variable
   soname_regexp = libefi(var|boot)\\.so\\..*
   symbol_name_regexp = ^(efidp_|efi_)[[:alnum:]_]+
-  change_kind = added-variable
-- 
2.17.1

